Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM ruby
+MAINTAINER Jonathan Claudius
+ENV PROJECT=github.com/mozilla/ssh_scan
+
+RUN mkdir /app
+ADD . /app
+WORKDIR /app
+
```

Comment:
Unfortunately, the `ruby` container doesn't have `rvm` installed.
`/bin/sh: 1: rvm: not found`